ui/src/i18n/pl.json

```diff
@@ -307,7 +307,7 @@
   "settings.reset.help": "Jeśli masz problemy z podłączeniem Homebridge lub zewnętrznych akcesoriów takich jak kamery lub TV z HomeKit, być może będziesz musiał zresetować Homebridge.",
   "settings.service.debug_tooltip": "Niektóre wtyczki wymagają ustawienia zmiennej środowiskowej DEBUG, aby umożliwić rejestrowanie poziomu debugowania.",
   "settings.service.node_tooltip": "Opcje uruchomieniowe Node.js dla procesu Homebridge. To powinno być puste, chyba że wiesz, co robisz.",
-  "settings.startup.debug": "Tryb Homebridge Debug",
+  "settings.startup.debug": "Debugowanie Homebridge",
```

Comment:
This is a mode designation, not an action.

Reply:
It’s not a switch? which enable debug mode for homebridge.

Reply:
Yes, it is switch to enable mode, not to perform one time action.